### PR TITLE
Add missing themeable variables

### LINKED ISSUE #280: [terra-dev-site] Missing themeable variables
# Bug Report

The following themeable variables are missing:

**clinical-lowlight-theme**

--terra-dev-site-footer-background-color - lib/loader-components/ExampleTemplate.module.scss (line: 28, column: 4)
--terra-dev-site-header-background-color - lib/loader-components/ExampleTemplate.module.scss (line: 13, column: 4)
--terra-dev-site-marked-icon-link-background-image - lib/marked/marked.module.scss (line: 400, column: 4)
--terra-dev-site-template-content-background-color - lib/loader-components/ExampleTemplate.module.scss (line: 19, column: 4)

**orion-fusion-theme**

--terra-dev-site-blockquote-color - lib/mdx/MarkdownTags.module.scss (line: 28, column: 4)
--terra-dev-site-content-background-color - lib/wrappers/ContentLoaded.module.scss (line: 3, column: 4)
--terra-dev-site-footer-background-color - lib/loader-components/ExampleTemplate.module.scss (line: 28, column: 4)
--terra-dev-site-header-background-color - lib/loader-components/ExampleTemplate.module.scss (line: 13, column: 4)
--terra-dev-site-icon-link-background-image - lib/mdx/MarkdownTags.module.scss (line: 378, column: 4)
--terra-dev-site-layout-menu-panel-panel-background-color - lib/navigation/SecondaryNavigationLayout.module.scss (line: 22, column: 4)
--terra-dev-site-layout-menu-panel-panel-border-right - lib/navigation/SecondaryNavigationLayout.module.scss (line: 23, column: 4)
--terra-dev-site-marked-icon-link-background-image - lib/marked/marked.module.scss (line: 400, column: 4)
--terra-dev-site-template-content-background-color - lib/loader-components/ExampleTemplate.module.scss (line: 19, column: 4)

### PR DESCRIPTION
**Pull request by lokesh-0813**
### Summary

Added missing themeable variables.

Removed [Themed.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/terra-dev-site/ff/ima%20folder/ima%20nother%20folder/Themed.module.scss), as it was earlier used in [themed.ff.jsx](https://github.com/cerner/terra-dev-site/blob/main/src/terra-dev-site/ff/ima%20folder/ima%20nother%20folder/themed.ff.jsx). But it was changed in PR #245 and after that the CSS file is not being used anywhere in the repo.

Closes #280 

### Deployment Link

https://terra-dev-site-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: src/loader-components/clinical-lowlight-theme/ExampleTemplate.module.scss [hunk 1]
```diff
@@ -8,5 +8,8 @@
     --terra-dev-site-example-template-code-background-color: #1d2427;
     --terra-dev-site-example-template-keyboard-focus-outline: #fff;
     --terra-dev-site-example-template-code-toggle-color: #b2b5b6;
+    --terra-dev-site-footer-background-color: #004c76;
+    --terra-dev-site-header-background-color: #004c76;
+    --terra-dev-site-template-content-background-color: #1b2326;
```
Comment (emilyrohrbough) [inline src/loader-components/clinical-lowlight-theme/ExampleTemplate.module.scss:13]: Were these theme variables named correctly in the original style sheet? I would have expected these be prefixed ` --terra-dev-site-example-template-`?

File: themes/orion-fusion-theme/orion-fusion-theme.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
+
+:global {
+    --terra-dev-site-blockquote-color: #24292e;
+    --terra-dev-site-content-background-color: #fff;
```
Comment (emilyrohrbough) [inline themes/orion-fusion-theme/orion-fusion-theme.scss:5]: This should be added to [src/wrappers/orion-fusion-theme/ContentLoaded.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/wrappers/orion-fusion-theme/MarkdownWrapper.module.scss)

File: themes/orion-fusion-theme/orion-fusion-theme.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    --terra-dev-site-icon-link-background-image: inline-svg('<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi icon icon-link"><path d="M10.5 26.3a16 16 0 0 1 .9-5.3L8 17.6a6.73 6.73 0 0 1-2-4.8A6.41 6.41 0 0 1 8 8a6.76 6.76 0 0 1 9.6 0l9 9a6.88 6.88 0 0 1 0 9.6 6.27 6.27 0 0 1-3.4 1.8 2.65 2.65 0 0 0 .5.6l4.1 4.1a11.36 11.36 0 0 0 3.1-2.3 12.8 12.8 0 0 0 0-18.1l-9-9a12.63 12.63 0 0 0-9-3.7 12.63 12.63 0 0 0-9 3.7A12.1 12.1 0 0 0 0 12.8a12.63 12.63 0 0 0 3.7 9l7 7a21.05 21.05 0 0 1-.2-2.5zm27-4.6a16 16 0 0 1-.9 5.3l3.4 3.4a6.79 6.79 0 1 1-9.6 9.6l-9-9a6.88 6.88 0 0 1 0-9.6 6.27 6.27 0 0 1 3.4-1.8 2.65 2.65 0 0 0-.5-.6l-4-4a11.36 11.36 0 0 0-3.1 2.3 12.8 12.8 0 0 0 0 18.1l9 9a12.73 12.73 0 0 0 18-18l-7-7a8 8 0 0 1 .3 2.3z"/></svg>');
```
Comment (emilyrohrbough) [inline themes/orion-fusion-theme/orion-fusion-theme.scss:6]: This should be added to [src/mdx/orion-fusion-theme/MarkdownTags.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/mdx/orion-fusion-theme/MarkdownTags.module.scss)

File: themes/orion-fusion-theme/orion-fusion-theme.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    --terra-dev-site-layout-menu-panel-panel-background-color: #fff;
+    --terra-dev-site-layout-menu-panel-panel-border-right: #d3d4d5;
```
Comment (emilyrohrbough) [inline themes/orion-fusion-theme/orion-fusion-theme.scss:8]: These should be added to [src/navigation/orion-fusion-theme/CollapsingNavigationMenu.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/navigation/orion-fusion-theme/CollapsingNavigationMenu.module.scss)

File: themes/orion-fusion-theme/orion-fusion-theme.scss [hunk 5]
```diff
@@ -0,0 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
+
+:global {
+    --terra-dev-site-blockquote-color: #24292e;
```
Comment (emilyrohrbough) [inline themes/orion-fusion-theme/orion-fusion-theme.scss:4]: This should be added to [src/mdx/orion-fusion-theme/MarkdownTags.module.scss](https://github.com/cerner/terra-dev-site/blob/main/src/mdx/orion-fusion-theme/MarkdownTags.module.scss)

File: themes/orion-fusion-theme/orion-fusion-theme.scss [hunk 6]
```diff
@@ -0,0 +1,9 @@
+@import '~terra-mixins/lib/Mixins';
```
Comment (emilyrohrbough) [inline themes/orion-fusion-theme/orion-fusion-theme.scss:1]: This file can be removed.

Comment (pranav300) [reply themes/orion-fusion-theme/orion-fusion-theme.scss]: Removed it https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8

File: themes/orion-fusion-theme/orion-fusion-theme.scss (hunk 5 above)
Comment (pranav300) [reply themes/orion-fusion-theme/orion-fusion-theme.scss]: done here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8

File: themes/orion-fusion-theme/orion-fusion-theme.scss (hunk 4 above)
Comment (pranav300) [reply themes/orion-fusion-theme/orion-fusion-theme.scss]: updated here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8

File: themes/orion-fusion-theme/orion-fusion-theme.scss (hunk 3 above)
Comment (pranav300) [reply themes/orion-fusion-theme/orion-fusion-theme.scss]: updated here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8

File: src/loader-components/clinical-lowlight-theme/ExampleTemplate.module.scss (hunk 1 above)
Comment (pranav300) [reply src/loader-components/clinical-lowlight-theme/ExampleTemplate.module.scss]: updated here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8

File: themes/orion-fusion-theme/orion-fusion-theme.scss (hunk 2 above)
Comment (pranav300) [reply themes/orion-fusion-theme/orion-fusion-theme.scss]: Removed it here https://github.com/cerner/terra-dev-site/pull/291/commits/fb6ef1fd6c84755f6f3d70a272222bf029eb79c8

File: tests/wdio/reports/result-chrome-terra-dev-site.json [hunk 7]
```diff
@@ -0,0 +1,308 @@
+{
```
Comment (emilyrohrbough) [inline tests/wdio/reports/result-chrome-terra-dev-site.json:1]: this file should be deleted

File: tests/wdio/reports/result-clinical-lowlight-theme-chrome-terra-dev-site.json [hunk 8]
```diff
@@ -0,0 +1,308 @@
+{
```
Comment (emilyrohrbough) [inline tests/wdio/reports/result-clinical-lowlight-theme-chrome-terra-dev-site.json:1]: this should be deleted

File: tests/wdio/reports/result-de-chrome-terra-dev-site.json [hunk 9]
```diff
@@ -0,0 +1,308 @@
+{
```
Comment (emilyrohrbough) [inline tests/wdio/reports/result-de-chrome-terra-dev-site.json:1]: this should be deleted

File: tests/wdio/reports/result-orion-fusion-theme-chrome-terra-dev-site.json [hunk 10]
```diff
@@ -0,0 +1,308 @@
+{
```
Comment (emilyrohrbough) [inline tests/wdio/reports/result-orion-fusion-theme-chrome-terra-dev-site.json:1]: We prob need to add reports to the gitIgnore

File: tests/wdio/reports/result-chrome-terra-dev-site.json (hunk 7 above)
Comment (pranav300) [reply tests/wdio/reports/result-chrome-terra-dev-site.json]: Removed it here https://github.com/cerner/terra-dev-site/pull/291/commits/aee2cba7f1788c819e3abb3428f716dab55a0b2c.

File: tests/wdio/reports/result-de-chrome-terra-dev-site.json (hunk 9 above)
Comment (pranav300) [reply tests/wdio/reports/result-de-chrome-terra-dev-site.json]: Removed it here https://github.com/cerner/terra-dev-site/pull/291/commits/aee2cba7f1788c819e3abb3428f716dab55a0b2c.

File: tests/wdio/reports/result-clinical-lowlight-theme-chrome-terra-dev-site.json (hunk 8 above)
Comment (pranav300) [reply tests/wdio/reports/result-clinical-lowlight-theme-chrome-terra-dev-site.json]: Removed it here https://github.com/cerner/terra-dev-site/pull/291/commits/aee2cba7f1788c819e3abb3428f716dab55a0b2c.

File: tests/wdio/reports/result-orion-fusion-theme-chrome-terra-dev-site.json (hunk 10 above)
Comment (pranav300) [reply tests/wdio/reports/result-orion-fusion-theme-chrome-terra-dev-site.json]: Added here https://github.com/cerner/terra-dev-site/pull/291/commits/fc27778c1e6288dec7f38340407ded016fe10eb3

**Pull request merged**